Sources/RandomAccessCollection+Sweet.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+extension CollectionType {
+    func enumerateWithNext() -> [(Any, Any?)] {
```

Comment:
I haven't done that too, but maybe this helps https://github.com/Jasdev/Public-Extension/blob/master/PublicExtension.playground/Pages/CollectionType.xcplaygroundpage/Contents.swift